examples/air-quality-sensor-app/air-quality-sensor-common/air-quality-sensor-app.zap

```diff
@@ -4562,154 +4562,112 @@
               "reportableChange": 0
             },
             {
-              "name": "GeneratedCommandList",
-              "code": 65528,
-              "mfgCode": null,
-              "side": "server",
-              "type": "array",
-              "included": 1,
-              "storageOption": "External",
-              "singleton": 0,
-              "bounded": 0,
-              "defaultValue": null,
-              "reportable": 1,
-              "minInterval": 1,
-              "maxInterval": 65534,
-              "reportableChange": 0
-            },
-            {
-              "name": "AcceptedCommandList",
-              "code": 65529,
+              "name": "PeakMeasuredValue",
```

Comment:
(comment unrelated to the zap file, just needed somewhere to put it) Not sure if this is even compiled in the CI right now, but if you want to make sure the example stays conformance, I would suggest adding your example app to the CI and running the device conformance tests. 

see https://github.com/project-chip/connectedhomeip/blob/master/.github/workflows/tests.yaml#L505 for door lock